mne/label.py

```diff
@@ -1615,9 +1620,15 @@ def grow_labels(subject, seeds, extents, hemis, subjects_dir=None, n_jobs=1,
         labels = _grow_nonoverlapping_labels(subject, seeds, extents, hemis,
                                              vert, dist, names)
 
-    # add a unique color to each label
-    colors = _n_colors(len(labels))
-    for label, color in zip(labels, colors):
+    if colors is None:
+        # add a unique color to each label
+        label_colors = _n_colors(len(labels))
+    else:
+        # use specified colors
+        label_colors = np.empty((len(labels), 4))
+        label_colors[:] = colors
```

Comment:
`label_colors = np.array(colors)`

seems easier and equivalent.

also do you test somewhere that `len(colors) == len(labels)` ?

Reply:
No it's not equivalent. The point of doing the `np.empty` is that

1. It allows broadcasting like `colors=0.` if you don't care about the color they can all be black, or `color=(1., 0, 0)` to make them all red, and 
2. It automatically makes sure `len(labels) == len(colors)`

Reply:
ok then. +1 for MRG if CIs are green.

thx @olafhauk !

Reply:
```colors``` can also be of shape (, 4) (one color for all labels). I amended doc-string and added checks/tests.